implementation/contracts/DepositLog.sol

```diff
@@ -10,6 +12,10 @@ contract DepositLog {
       Everyone should be able to ENTIRELY rely on log messages
     */
 
+    // `TBTCDepositToken` mints a token for every new Deposit.
+    // If a token exists for a given ID, we know it is a valid Deposit address.
+    TBTCDepositToken tbtcDepositToken;
```

Comment:
🤔shadows `TBTCSystem.tbtcDepositToken`. will raise a `DeclarationError` with solc >= 0.6 due to re-declaration in same class.